nfs_client.yml

```diff
@@ -17,6 +17,7 @@
 barclamp:
   name: 'nfs_client'
   display: 'NFS Client'
+  description: 'NFS Mount Configuration'
```

Comment:
I currently see "Setup of NFS mounts" in the crowbar webui, which I slightly prefer to the above.